<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg</class>
 <widget class="QWidget" name="dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dlg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;主机(&lt;span style=&quot; text-decoration: underline;&quot;&gt;H&lt;/span&gt;):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="m_host">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>2</width>
         <height>1</height>
        </size>
       </property>
       <property name="text">
        <string>127.0.0.1</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;端口(&lt;span style=&quot; text-decoration: underline;&quot;&gt;P&lt;/span&gt;):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="m_port">
       <property name="text">
        <string>3600</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;用户名(&lt;span style=&quot; text-decoration: underline;&quot;&gt;U&lt;/span&gt;):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="m_name">
       <property name="text">
        <string>test</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;密码(&lt;span style=&quot; text-decoration: underline;&quot;&gt;P&lt;/span&gt;):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="m_password">
       <property name="text">
        <string>test</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="m_connectButton">
       <property name="text">
        <string>连接</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="m_connectType">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string/>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="m_refreshButton">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
      <widget class="QPushButton" name="m_disconnectButton">
       <property name="text">
        <string>断开</string>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>本地</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLabel" name="label_5">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>站点：</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="m_localFolder">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>100</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
           <widget class="QPushButton" name="m_uploadButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>上传</string>
            </property>
           </widget>
           <widget class="QPushButton" name="m_localDeleteButton">
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </widget>
          <widget class="QTableView" name="m_localView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>100</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>服务器</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QSplitter" name="splitter_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLabel" name="label_6">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>站点：</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="m_serverFolder">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>100</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
           <widget class="QPushButton" name="m_downloadButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>下载</string>
            </property>
           </widget>
           <widget class="QPushButton" name="m_mkdirButton">
            <property name="text">
             <string>创建</string>
            </property>
           </widget>
           <widget class="QPushButton" name="m_deleteButton">
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </widget>
          <widget class="QTableView" name="m_serverView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>100</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTextEdit" name="m_log">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
